net/packet/packet.go

```diff
@@ -18,7 +18,7 @@ import (
 const unknown = ipproto.Unknown
 
 // RFC1858: prevent overlapping fragment attacks.
-const minFrag = 60 + 20 // max IPv4 header + basic TCP header
+const minFragOct = (60 + 20) / 8 // max IPv4 header + basic TCP header in octets
```

Comment:
I agree that there is a problem, but this doesn't look right. It says the IPv4+TCP header is 10 octets, which is not true.

Reply:
True, the terminology bug starts in the RFC. I'll change it to "fragment blocks".